sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" />
-    <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="NetStandard.Library" VersionOverride="2.0.3"/>
```

Comment:
Why are we adding new dependencies?

Reply:
They aren't new dependencies just dependency closures.